numba/core/ir.py

```diff
@@ -336,6 +336,12 @@ def _rec_list_vars(self, val):
         else:
             return []
 
+    def __repr__(self):
+        return self.__str__()
+
+    def __str__(self):
+        return super().__repr__()
+
```

Comment:
My [suggestion](https://github.com/numba/numba/pull/9521#discussion_r1559923460) is leading to other test failures since the string repr behavior changed. @DrTodd13, maybe it is easier just to remove the these changes all together, assuming it is just needed for debugging.

Reply:
Done. @sklam